### PR TITLE
Removing extra spaces

### PR DESCRIPTION
**Pull request by Starnol**
and trying to add the files we went over Friday

### CODE REVIEW

**Comment by Starnol** [comment]
ready for review.

**Comment by bjones1** [comment]
Should I review this PR, or the other ones you opened? Close everything that's old so I know what to look at.

**Comment by Starnol** [comment]
I have closed the other ones. And left comments. Alot of your comments were about the given code of the game. I am new to python so I can't really explain their given code. I had two thought processes during my second try with the game.  You originally asked for our thought processes. You are seeing inside my head.

File: OMC56/Thor Method.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+# 
+# |
+# |
+#
+# Provided Code
+# ------------------
+# This game has a built in code for the directions that you can't see. After trail and error, I figured out that all it is doing is either increasing/decreasing x and/or y. It is **not** updating Thor's position. Which can cause him to go outside of the boundry. 
+#
+
+from re import X
```
Comment (bjones1) [inline OMC56/Thor Method.py:23]: I don't think you use this -- remove it if it's not used.

File: OMC56/Thor Method.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+import sys
+import math
+
+# *Auto- generated code below aims to helping you parse the standard input according to the problem statement. Hint: you can use the debug stream to print initialTX and initialTY, if Thor seems not to follow your instructions.*
+ # **Light_x**: the X position of the light of power
```
Comment (bjones1) [inline OMC56/Thor Method.py:28]: Format as bulleted list or definition list:

```
#   :Light_x: the X position of the light of power
#   :Light_y: the Y position of the light of power
```

File: OMC56/Thor Method.py [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+ #
+ # **Initial_tx**: Thor's starting X position
+ #
+ # **Initial_ty**: Thor's starting Y position
+ #
+light_x, light_y, initial_tx, initial_ty = [int(i) for i in input().split()]
+
+# game loop
+while True:
+    remaining_turns = int(input())  # The remaining amount of turns Thor can move. Do not remove this line.
```
Comment (bjones1) [inline OMC56/Thor Method.py:40]: Put the comment on the previous line so CodeChat can render it.

**Review by bjones1 — CHANGES_REQUESTED** [review]

File: OMC56/Thor Method_w_JP_comments.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
+# ************************
```
Comment (bjones1) [inline OMC56/Thor Method_w_JP_comments.py:1]: Combine comments from JP so there's just one `Thor method.py` file will all improvements made to it. Delete this file.

**Review by bjones1 — CHANGES_REQUESTED** [review]

**Comment by bjones1** [comment]
Liv, you need to address the comments I've left here before your homework assignment is complete.

**Comment by Starnol** [comment]
I will work on this after work today. I want to get this cleaned up now that I believe I understand how smart git works. I would also like to set up a time to talk about how the framework looks for the microprocessor ebook. That would be a good guideline to work with. I have two Zeros with github. I was able to get the pull request for the first one to work. I will be trying to merge the files/commits this evening.

Very Respectfully,
Olivia Starnes
________________________________

**Comment by bjones1** [comment]
I look forward to seeing your progress! Kaitlyn scheduled a meeting today at 4:00 PM about the e-book. I've also created a channel in the class MS Teams account for the e-book and discussion on it.

**Comment by bjones1** [comment]
Ping...still waiting on this...

**Comment by Starnol** [comment]
My smart git is longer working. It keeps saying I need a License.
________________________________

**Comment by bjones1** [comment]
You may have signed up for a commercial license with a 30-day trial. Check the smartgit website or reinstall to fix. Pick the free / open source license.

**Pull request closed without merging**